packages/node_modules/@ciscospark/redux-module-media/src/actions.js

```diff
@@ -202,17 +202,29 @@ export function listenForIncomingCalls(sparkInstance, locusUrl) {
 
       return incomingCall.acknowledge()
         .then(() => {
-          const removeIncoming = () => dispatch(removeCall(incomingCall, incomingCall.locus.url));
+          const removeIncoming = () => {
+            incomingCall.off();
```

Comment:
This seems to remove all listeners from the incoming call. Would this affect the bricklets in recents? i.e. when a call is rejected, it should still appear as an active call in the recents list.

Reply:
The only time it calls `removeIncoming` is when the call goes inactive. We can probably remove the `.off` line if it is a concern, but it is immediately followed by the `removeCall` which removes it completely from our store anyway.

Reply:
Gotcha, yea I may have read this before I saw the rest. I think it's more of a concern for the `membership:declined` event where we directly call `removeIncoming()`.